Enhance client side throttling information (#2739)
kubernetes-sigs/aws-load-balancer-controller: docs/deploy/configurations.md

* you can no longer alter the value of an `alb.ingress.kubernetes.io/group.name` annotation on an existing Ingress.

###  throttle config

The default throttle config used is.

```
WAF Regional:^AssociateWebACL|DisassociateWebACL=0.5:1,WAF Regional:^GetWebACLForResource|ListResourcesForWebACL=1:1,WAFV2:^AssociateWebACL|DisassociateWebACL=0.5:1,WAFV2:^GetWebACLForResource|ListResourcesForWebACL=1:1
```

> Client side throttling enable gradual scaling of the api calls. Additional throttle config can be specified by `--aws-api-throttle` flag. The ServiceId used in `--aws-api-throttle` The serviceID can be obtained from the API definition in AWS SDK. For e.g, ELBv2 it is [Elastic Load Balancing v2](https://github.com/aws/aws-sdk-go/blob/main/models/apis/elasticloadbalancingv2/2015-12-01/api-2.json#L9).

Reviewer: ```suggestion
Client side throttling enables gradual scaling of the api calls. Additional throttle config can be specified via the `--aws-api-throttle` flag. You can get the ServiceID from the API definition in AWS SDK. For e.g, ELBv2 it is [Elastic Load Balancing v2](https://github.com/aws/aws-sdk-go/blob/main/models/apis/elasticloadbalancingv2/2015-12-01/api-2.json#L9).
```
